### ISSUE TITLE
html-has-lang and iframe-has-title false warnings

### ISSUE THREAD

**Issue by EvgenyOrekhov** [brendanmorrell/eslint-plugin-styled-components-a11y#54]
When I have a styled component that wraps another styled component from a different file, I get these warnings:

```txt
<html> elements must have the lang prop. eslint(styled-components-a11y/html-has-lang)
<iframe> elements must have a unique title property. eslint(styled-components-a11y/iframe-has-title)
```

Example code:

index.tsx

```ts
import React from 'react';
import { Link } from 'components/Link';
import { styled } from '@mui/material/styles';

export const Example = () => {
  // The following line gives warnings for StyledLink:
  return <StyledLink href='https://example.com'>foo</StyledLink>;
};

const StyledLink = styled(Link)`
  padding: 0;
`;
```

components/Link.tsx

```ts
import React from 'react';
import { styled } from '@mui/material/styles';

export const Link = styled('a')`
  font-size: 13px;
`;
```

This started happening with 2.1.0, worked fine with 2.0.0.

**Comment by HaleyWardo** [comment]
Seeing the same thing as well. Hopefully this can be resolved soonish so I can start using it. 🙏

**Comment by shroy** [comment]
I'm experiencing this as well when wrapping a dot-notated component. 

```jsx
const TileBody = styled(TileBody.Right)`
   //....
`;
```

**Comment by brendanmorrell** [comment]
The dot notation is also where i see it. Ill try and take a look this
weekend. Thanks for the info

**Comment by brendanmorrell** [comment]
Alright. Seemed there were two cases were the rules were misfiring: When the component is one where the tag is a key on another object, such as in `styled(animated.div)` and in cases where we can't determine the base tag of a custom component, such as when importing a component from another file and calling `styled(CustomComponent)` on it. I believe I now have solved both cases and I am not getting the error anywhere anymore. [Version 2.1.3 ](https://www.npmjs.com/package/eslint-plugin-styled-components-a11y/v/2.1.3)is up now. @EvgenyOrekhov, @shroy, @HaleyWardo: If any of you are still seeing the error anywhere, if you create a reproduction in a codesandbox, I can take another look. cheers!

**Comment by HaleyWardo** [comment]
It's working perfectly! Thank you for the quick fix @brendanmorrell! 👏

**Comment by EvgenyOrekhov** [comment]
It works now, thank you! 🎉

**Comment by shroy** [comment]
Thanks for the quick fix! It's looking good!

**Comment by brendanmorrell** [comment]
Great! glad to hear it. Thanks, guys
